lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1654,7 +1659,7 @@ Attribute AMDMfmaEncodingAttr::parse(AsmParser &parser, Type type) {
 
   return parser.getChecked<AMDMfmaEncodingAttr>(
       parser.getContext(), versionMajor, versionMinor, warpsPerCTA,
-      instrShape[0], instrShape[1], isTransposed, *CTALayout);
+      instrShape[0], instrShape[1], kDim, isTransposed, *CTALayout);
```

Comment:
Can we just bump the instrShape size (2->3) and set a default value of 1 for the 3 dim?